FRAuth/FRAuth/Callbacks/WebAuthnRegistrationCallback.swift

```diff
@@ -282,9 +284,12 @@ open class WebAuthnRegistrationCallback: WebAuthnCallback {
     /// Registers against AM's `WebAuthn Registration Node` based on the JSON callback and WebAuthn's properties within the Callback
     /// - Parameters:
     ///   - node: Optional `Node` object to set WebAuthn value to the designated `HiddenValueCallback`
+    ///   - deviceName: Optional `Device Name` object to set Device Name value to the designated `HiddenValueCallback`
     ///   - onSuccess: Completion callback for successful WebAuthn assertion outcome; note that the outcome will automatically be set to the designated `HiddenValueCallback`
     ///   - onError: Error callback to notify any error thrown while generating WebAuthn assertion
-    public func register(node: Node? = nil, onSuccess: @escaping StringCompletionCallback, onError: @escaping ErrorCallback) {
+    public func register(node: Node? = nil, deviceName: String? = nil, onSuccess: @escaping StringCompletionCallback, onError: @escaping ErrorCallback) {
+        
+        self.deviceName = deviceName
```

Comment:
Do we need an instance variable? Can we just treat it as local variable in the function?

Reply:
I see no problem with having a public var so that the device name can be accessed later on in the WebAuthnRegistration lifetime. Not required but it I would leave it in, unless we get any major objections

Reply:
If not necessary, I would prefer to remove it. The following sequence of code may break developer expectations:
```
WebAuthnRegistrationCallback.register(<without device name>)
WebAuthRegistrationCallback.deviceName = "MyDeviceName"
node.next() // Proceed to next but the device name is not set.
```

Reply:
gotcha, will fix it

Reply:
if you really want the deviceName accessed later,  we can make  private set and public get